arch/arm/core/aarch64/Kconfig

```diff
@@ -56,6 +56,13 @@ config CMSIS_V2_THREAD_DYNAMIC_STACK_SIZE
 config IPM_CONSOLE_STACK_SIZE
 	default 2048
 
+config IMAGE_HEADER
```

Comment:
this appears very specific for aarch64, should it be named: `AARCH64_IMAGE_HEADER` or `ARM64_IMAGE_HEADER` instead ?